docs/source/howto/codes.rst

```diff
@@ -45,8 +45,92 @@ Design guidelines
 Running external codes
 ======================
 
-`#3987`_
+The process of running an external code through AiiDA involves loading and instantiating the appropriate ``CalcJob`` wrapping class for the code, filling the inputs necessary, and then running or submitting the constructed object.
```

Comment:
"The process of running an external code .... involves" is pretty verbose language for a how-to.
How about something more direct and to the point:
> To run an external code with AiiDA, you need a :ref:`plugin <correct link to more info>` for it.
> The plugin's job is to transform the input *nodes* to the input files that the code expects.
> If the plugin for your code already exists (see the `plugin registry <http://registry>`_ what is available) and you have installed the :ref:`plugin package <ref>`, you can load it using the ``CalculationFactory``.